### DISCUSSION TITLE
Bugs: syncing and editing

### DISCUSSION THREAD

**Discussion by Brog33** [bitfireAT/icsx5#204] [General]
After running ICSx5 (2.2-beta.1-standard) for quite a while now with minimal problems, it has started flooding my notification area with sync errors. This has been going on for a few days now.

So, when I go into the app and refresh the subscriptions manually, the first entry fails and all the subsequent are synced OK. - But when the scheduled sync is running, every subscription fail (!). 
Why could this subscription be failing then - the URL works and it seems to load in other apps: URL: ics.fixtur.es/v2/league/champions-league.ics

Another issue is that when i tap on any of the subscriptions (to edit), the app crashes (just closes down without any error requester). Is there a logfile or anything else I could provide?

![Screenshot_2024-02-05-07-40-28-24_722202591f00e37161fc029436fa2e48](https://github.com/bitfireAT/icsx5/assets/1303537/598eae6a-5e1a-4f36-a212-91905c2674c2)

**Comment by sunkup** [comment]
Hey, please try out the newest version.
